solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -1141,11 +1141,11 @@
   <updateProcessor class="solr.ParseDoubleFieldUpdateProcessorFactory" name="parse-double"/>
   <updateProcessor class="solr.ParseDateFieldUpdateProcessorFactory" name="parse-date">
     <arr name="format">
-      <str>yyyy-MM-dd'T'HH:mm[:ss[.SSS]][z</str>
```

Comment:
You just changed the default date patterns to not parse patterns it parsed before.  We need to be cautious!  For example it used to parse a comma to separate the milliseconds.  And the 'T' was optional.

Whatever patterns exist here ought to be identical in documentation & test configs (which yes is annoying to keep in sync but shouldn't change often).

Reply:
So should I keep the old patterns in addition to the ones from SOLR-12591?

Reply:
By "the old patterns" do you mean the _current_ patterns we see in the default config?  Yes.  You can tweak an existing pattern to parse more things, but don't remove the ability to parse something it would have parsed before!  And yes add/merge the additional ones from SOLR-12591.  By "merge" I mean modify some existing patterns to match some of the ones in SOLR-12591.